docs/guides/workflow-syntax.rst

```diff
@@ -59,9 +59,9 @@ Passing outputs from one task as inputs to another one
 Returning workflow outputs
 --------------------------
 
-This designates some of the workflow variables to be returned via ``orq get workflow-run-results``.
+Any variable returned by the workflow function is a workflow result, and will be returned via ``orq get workflow-run-results``.
```

Comment:
**suggestion**: it's unrelated to this PR, but I've noticed the CLI invocation is out of date

```suggestion
Any variable returned by the workflow function is a workflow result, and will be returned via ``orq wf results``.
```